code/game/objects/items/clown_items.dm

```diff
@@ -186,12 +186,13 @@
 	attack_verb_continuous = list("HONKS")
 	attack_verb_simple = list("HONK")
 	///sound file given to the squeaky component we make in Initialize()
-	var/soundfile = 'sound/items/bikehorn.ogg'
+	var/sound_file = 'sound/items/bikehorn.ogg'
 
 /obj/item/bikehorn/Initialize()
 	. = ..()
-	//LoadComponent so child types dont stack squeak components
-	LoadComponent(/datum/component/squeak, list(soundfile=1), 50, 0, 0, 0, 0, 20, 0)
+	var/list/sound_list = list()
+	sound_list[sound_file] = 1
+	AddComponent(/datum/component/squeak, sound_list, falloff_exponent = 20)
```

Comment:
```suggestion
	AddComponent(/datum/component/squeak, list((soundfile) = 1), 50, falloff_exponent = 20)
```
You can use parentheses to ensure it's parsed as a variable instead of a string.

Reply:
oops, i thought i carried it over (also wtf byond)

Reply:
feexed

Reply:
maintainers dont like it 😔